news/_posts/2023-01-03-178-lts-80.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "PrestaShop 1.7.8 is now a LTS"
+subtitle: "What to expect for PrestaShop 1.7 and 8"
+date:   2023-01-03
+authors: [ MathieuFerment ]
+icon: icon-code
+tags:
+ - version
+ - releases
+ - 1.7.8.x
+ - 8.0.x
+icon: icon-warning
+---
+
+Now that PrestaShop 8.0 is out, here is a focus on maintained branches of the PrestaShop project and what to expect.
+
+## Reminder: PrestaShop branching strategy
+
+In [August 2015](https://build.prestashop-project.org/news/2015/introducing-new-branching-model-prestashop/) PrestaShop adopted a new branching model for the project.
+
+The model was refined over the years, for example with [this page on the developer documentation](https://devdocs.prestashop-project.org/8/contribute/contribution-guidelines/supported-branches/#bug-fixes-and-patch-branches) about what bugs do qualify for a patch version or the [Backward Compatibility Promise](https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md) that defines what can changes can be introduced in a version, but we can say the model has lived quite well as the project is still following it in 2023.
+
+## PrestaShop branches in 2023
+
+As of today, January 2023, what are the maintained branches and how do they comply with this model?
+
+On [PrestaShop main repository](https://github.com/prestashop/prestashop) three branches are maintained
+
+### 1.7.8.x
+
+Branch `1.7.8.x` is the branch that gave birth to PrestaShop 1.7.8.0 and the patches that followed. As of today last 1.7.8 version is [PrestaShop patch release 1.7.8.8](https://build.prestashop-project.org/news/2022/prestashop-1-7-8-8-maintenance-release/).
+
+Following PrestaShop 8.0.0 release, the `1.7.8.x` branch has entered security-only maintenance phase. As detailed in [the article published in December 2021](https://build.prestashop-project.org/news/2021/prestashop-beyond-1-7/), this means that patch releases of PrestaShop 1.7.8 will only be published if critical bugs are reported or if security fixes are needed.
+
+As a consequence, please do not open a pull request against this branch unless it is a bugfix that solves a critical bug or a security fix. Other Pull Requests will not be accepted.
+
+This maintenance period will end when PrestaShop 9.0.0 is released. When this day comes PrestaShop 1.7 will not be maintained anymore.
+
+### 8.0.x
+
+Branch `8.0.x` is the branch that gave birth to PrestaShop 8.0.0 and the patches that will follow. It was branched out from `develop` branch when [PrestaShop 8.0 freeze](https://build.prestashop-project.org/news/2022/prestashop-feature-freeze-v8/) happened in June 2022.
```

Comment:
```suggestion
Branch `8.0.x` is the branch that gave birth to PrestaShop 8.0.0 and the patches that followed. It was branched out from `develop` branch when [PrestaShop 8.0 freeze](https://build.prestashop-project.org/news/2022/prestashop-feature-freeze-v8/) happened in June 2022.
```

Reply:
No, there is no 8.0 patches today :) . The first patch will be 8.0.1 and it is not released yet. So it is "will follow" ;)